numpy/lib/type_check.py

```diff
@@ -492,7 +492,8 @@ def _real_if_close_dispatcher(a, tol=None):
 @array_function_dispatch(_real_if_close_dispatcher)
 def real_if_close(a, tol=100):
     """
-    If complex input returns a real array if complex parts are close to zero.
+    If input is complex with all complex parts close to zero, return real
```

Comment:
General approach to rewording seems fine, but the second "complex" should be "imaginary", i.e.

> If input is complex with all imaginary parts close to zero, return real parts.